.gitignore

```
@@ -1,2 +1,3 @@
-.DS_Store
-
+*
+!README.md
+!forkbomb.c
```

Please see [3.206 Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)

> ## 3.206 Line  
> A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

```suggestion
!forkbomb.c
```